### PR TITLE
add og template

### LINKED ISSUE #39: include opengraph tags in site pages
- opengraph tags can probably be global and be included in a base template that all other templates use
- take tags from nyc-noise.com

### PR DESCRIPTION
**Pull request by evanconway**
resolves #39 
Just adds a template including the meta:og tags in the current nyc-site to the main page.

### CODE REVIEW

**Comment by gregsadetsky** [comment]
absolutely great, thank you!

do you mind downloading the image referenced in `og:image` and serving it from the static directory? that way, once we transition the site over, we won't be referencing an image that doesn't exist anymore :-)

thanks!

**Comment by gregsadetsky** [comment]
+ same point as above for the twitter image og tag :-)

**Comment by gregsadetsky** [comment]
thanks for the change re: static image!

verrrrrrrry small (and annoying/django-y) thing -> could you please move the image to core/static/core ? I know it's weird to have "core" appear twice (i.e. have a directory inside of static called the same as the app "above") but that's what django recommends/discusses in its docs:

https://docs.djangoproject.com/en/4.2/howto/static-files/#configuring-static-files

scroll down specifically to "Static file namespacing"

TLDR move the image to core/static/core, and update the reference in the html file to core/images/...

thanks again!

**Comment by evanconway** [comment]
Was able to test using ngrok and https://www.opengraph.xyz. Unfortunately the meta tags still appear broken on many og testing sites, and linking on discord does not appear correctly. I believe this is because ngrok issues a warning page that must be clicked through before the regular content can be viewed, which would break most of these things. But I'm not 100% positive.

**Comment by gregsadetsky** [comment]
there's currently a problem in [opengraph.html](https://github.com/gregsadetsky/nycnoise/pull/76/files#diff-fc086058dd4fbf8035e4093914db49ebc8c2681e783b4db50eecfb5ed6245f25) that's due to lines like these:

`<meta name="description" content={% meta_description %}>`

what's returned by the template tag will be inserted into the html _as is_, without any quote signs. view the source of http://localhost:8000/ to see it yourself -- you'll see this kind of html:

```html
<meta name="description" content=NYC experimental live music calendar • noise, improv, jazz, new music, avant-electronics, weirdos • performance against corporate interests!>
```

which is not valid, as it should be `content="...."`

it's the same problem for all of the `content=` attributes that are calling on a template tag.

let me know if you'd like to fix this (happy to take it on as well if it can help)

**Comment by evanconway** [comment]
Missing quotes in opengraph.html should be fixed now.

**Review by gregsadetsky — APPROVED** [review]
:shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit::shipit:
